### ISSUE TITLE
Backprop generator is missing data files

### ISSUE THREAD

**Issue by rdadolf** [breagen/MachSuite#6]
Our old data generator used rather lengthy headers for specifying training data and labels. We removed them, but the generator code still references their variables.

The right thing to do is create a synthetic training set by adding noise to a computable function. This would fix the undefined variable problems, and it would also enable `backprop/backprop` to work properly when the `input_dimension` and `possible_outputs` parameters are changed.

**Comment by johnnysDeu** [comment]
hello

is there any chance that you update a working version of backprop? i am a student and i need it for a project
thanks a lot

**Comment by rdadolf** [comment]
Hi @johnnysDeu. Just to be clear about this, backprop itself does work, it's just the generator which is affected by this. Moreover, it's not too terribly tough to modify backprop to use a different neural net topology, the bug just means that MachSuite won't auto-generate a set of verification outputs for you to check against.

At the moment, this isn't on the top of our priority list, but if it's becoming a roadblock for more users, then we can revisit it.

**Comment by hkultala** [comment]
I am also affected by this. I need to do some porting because my platform does not support file IO, so I cannot use the ready data files, I need to generate the data into headers and link it inside the binary.
